frontend/pages/index.vue

```diff
@@ -1,14 +1,275 @@
 <template>
-    <div class="home">
+  <div class="homepage">
+    <!-- Obere Sektion mit blauem Hintergrund -->
+    <div class="intro-section">
```

Comment:
I see a slight white line above this that is not continous. Not sure why it is there but it does not look right to me.... 
![image](https://github.com/user-attachments/assets/921e05ca-bb10-4f3b-b875-2bb191a284ec)